src/persistence.js

```diff
@@ -326,15 +332,12 @@ export default function _Persistence(mpInstance) {
         return parsedDecodedProducts;
     };
 
-    // https://go.mparticle.com/work/SQDSDKS-6021
-    this.setLocalStorage = function() {
+    this.setProductStorage = function() {
```

Comment:
What is the reason for doing product changes in this PR?  Seems out of scope, but maybe I'm missing something.